yarn/src/main/scala/org/apache/spark/deploy/yarn/ClientArguments.scala

```diff
@@ -73,12 +73,12 @@ private[spark] class ClientArguments(args: Array[String], sparkConf: SparkConf)
       .orNull
     // If dynamic allocation is enabled, start at the max number of executors
     if (isDynamicAllocationEnabled) {
-      val maxExecutorsConf = "spark.dynamicAllocation.maxExecutors"
-      if (!sparkConf.contains(maxExecutorsConf)) {
+      val minExecutorsConf = "spark.dynamicAllocation.maxExecutors"
+      if (!sparkConf.contains(minExecutorsConf)) {
```

Comment:
so the ExecutorAllocationManager already validates that minExecutorConf exists and is valid (not negative and non-zero). Do we need this? or are we being paranoid? If so, maybe an assertion is better?